src/main/java/com/amazon/opendistroforelasticsearch/security/securityconf/impl/v6/InternalUserV6.java

```diff
@@ -79,6 +79,12 @@ public String getHash() {
         public void setHash(String hash) {
             this.hash = hash;
         }
+        /**
```

Comment:
nit: add empty line before and after.

Reply:
Done.

Reply:
Sorry, by "after" I mean to add empty line after the method, so one method is separated from another.

Reply:
Delete '/** .. */` comment (it uses doc style comment, and it should not be doc). It should be sufficient to have a single line comment inside `setPassword()` method body.